cg/store/crud/read.py

```diff
@@ -1463,3 +1468,14 @@ def get_cases_for_sequencing_qc(self) -> list[Case]:
                 CaseFilter.HAS_SEQUENCE,
             ],
         ).all()
+
+    def get_case_ids_with_sample(self, sample_id: int) -> list[str]:
+        """Return all case ids with a sample."""
+        sample: Sample = self.get_sample_by_entry_id(sample_id)
+        return [link.case.internal_id for link in sample.links] if sample else []
+
+    def get_case_internal_ids_with_samples(self, sample_ids: list[int]) -> list[str]:
```

Comment:
these methods should either call the internal_ids case_ids or case_internal_ids but they should not be mixed.

Reply:
Fixed in https://github.com/Clinical-Genomics/cg/pull/3507/commits/3c8bc540c46fee39b2f6b092705b87d9da440591